src/components/MegadraftEditor.js

```diff
@@ -185,7 +185,25 @@ export default class MegadraftEditor extends Component {
   }
 
   render() {
-    const {editorState, stripPastedStyles, spellCheck} = this.props;
+    const {
```

Comment:
How about using using the [spread operator](https://facebook.github.io/react/docs/jsx-in-depth.html#spread-attributes) to pass props to the `Editor` instance?

This way we don't have to map every single supported prop.

We just have to avoid overriding props handled by `MegadraftEditor` like `blockRendererFn`, `handleReturn` and others...